Rule-008 Tests.postman_collection.json

```diff
@@ -0,0 +1,2280 @@
+							"                    }\r",
+							"                }\r",
+							"            },\r",
+							"            \"Cdtr\": {\r",
+							"                \"Nm\": \"Felicia Easton Quill\",\r",
+							"                \"Id\": {\r",
+							"                    \"PrvtId\": {\r",
+							"                        \"DtAndPlcOfBirth\": {\r",
+							"                            \"BirthDt\": \"1935-05-08\",\r",
+							"                            \"CityOfBirth\": \"Unknown\",\r",
+							"                            \"CtryOfBirth\": \"ZZ\"\r",
+							"                        },\r",
+							"                        \"Othr\": {\r",
+							"                            \"Id\": \"+27707650428\",\r",
+							"                            \"SchmeNm\": {\r",
+							"                                \"Prtry\": \"MSISDN\"\r",
+							"                            }\r",
+							"                        }\r",
+							"                    }\r",
+							"                },\r",
+							"                \"CtctDtls\": {\r",
+							"                    \"MobNb\": \"+27-707650428\"\r",
+							"                }\r",
+							"            },\r",
+							"            \"CdtrAcct\": {\r",
+							"                \"Id\": {\r",
+							"                    \"Othr\": {\r",
+							"                        \"Id\": \"352afab0eed3e035ff73cec42277d4a9f66f289cee9c16dc6bca02eba4d91566\",\r",
+							"                        \"SchmeNm\": {\r",
+							"                            \"Prtry\": \"PSEUDO\"\r",
+							"                        }\r",
+							"                    }\r",
+							"                },\r",
+							"                \"Nm\": \"Felicia Quill\"\r",
+							"            },\r",
+							"            \"Purp\": {\r",
+							"                \"Cd\": \"MP2P\"\r",
+							"            }\r",
+							"        },\r",
+							"        \"RgltryRptg\": {\r",
+							"            \"Dtls\": {\r",
+							"                \"Tp\": \"BALANCE OF PAYMENTS\",\r",
+							"                \"Cd\": \"100\"\r",
+							"            }\r",
+							"        },\r",
+							"        \"RmtInf\": {\r",
+							"            \"Ustrd\": \"Payment of USD 30713.75 from April to Felicia\"\r",
+							"        },\r",
+							"        \"SplmtryData\": {\r",
+							"            \"Envlp\": {\r",
+							"                \"Doc\": {\r",
+							"                    \"Xprtn\": \"2021-11-30T10:38:56.000Z\"\r",
+							"                }\r",
+							"            }\r",
+							"        }\r",
+							"    }\r",
+							"}]\r",
+							"\r",
+							"const transactionGraph = [\r",
+							"{\r",
+							"    _key: \"d923682a97089a841615181e58355b8eeee108007f7898\",\r",
+							"    _from: \"accounts/24058ebbdd500fe2307cdd59186e0abcd9726de434ae492dbe9d1e12822914aa\",\r",
+							"    _to: \"accounts/352afab0eed3e035ff73cec42277d4a9f66f289cee9c16dc6bca02eba4d91566\",\r",
+							"    Amt: 2000,\r",
+							"    TxTp: \"pacs.008.001.10\",\r",
+							"    TxSts: \"ACCC\",\r",
```

Comment:
Welldone @vorsterk on this postman tests. Why does your Pacs008 edges have Txsts????